### PR TITLE
Update MemSQL requirements

### PR DESCRIPTION
**Pull request by Jessie212**

### CODE REVIEW

File: docs/src/main/sphinx/connector/memsql.rst [hunk 1]
```diff
@@ -7,6 +7,14 @@ creating tables in an external SingleStore database. The SingleStore connector
 is very similar to the MySQL connector with the only difference being the
 underlying driver.
 
+Requirements
+------------
+
+To connect to SingleStore, you need:
+
+* SingleStore version 7.3 or higher.
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/memsql.rst:15]: 3.0.0 or higher

Based on default tag in https://github.com/trinodb/trino/blob/master/plugin/trino-memsql/src/test/java/io/trino/plugin/memsql/TestingMemSqlServer.java#L33

File: docs/src/main/sphinx/connector/memsql.rst [hunk 2]
```diff
@@ -7,6 +7,14 @@ creating tables in an external SingleStore database. The SingleStore connector
...
+* Network access from the Trino coordinator and workers to SingleStore.
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/memsql.rst:16]: Add info about port

**Review by mosabua — CHANGES_REQUESTED** [review]

**Review by mosabua — APPROVED** [review]

File: docs/src/main/sphinx/connector/memsql.rst (hunk 1 above)
Comment (hashhar) [reply docs/src/main/sphinx/connector/memsql.rst]: That tag structure is a bit misleading. The MemSQL version is 7.1.13 in that tag. The suffix is the MemSQL Cluster-in-a-box tooling version - which is unrelated to MemSQL version being used.

**Review by m57lyra — APPROVED** [review]
LGTM

**Review by hashhar — COMMENTED** [review]
@Jessie212 Please revert the version change to 7.3. It looks good to be merged otherwise.

See https://github.com/trinodb/trino/pull/8305#discussion_r653904376

**Comment by Jessie212** [comment]
This branch has two commits. Closing and starting over.

**Pull request closed without merging**